securedrop/tests/utils/db_helper.py

```diff
@@ -81,31 +83,31 @@ def mock_verify_token(testcase):
 def mark_downloaded(*submissions):
     """Mark *submissions* as downloaded in the database.
 
-    :param db.Submission submissions: One or more submissions that
+    :param models.Submission submissions: One or more submissions that
                                       should be marked as downloaded.
     """
     for submission in submissions:
         submission.downloaded = True
-    db.db_session.commit()
+    models.db.session.commit()
 
 
-# db.{Source,Submission}
+# models.{Source,Submission}
 
 def init_source_without_keypair():
     """Initialize a source: create their database record and the
     filesystem directory that stores their submissions & replies.
     Return a source object and their codename string.
 
-    :returns: A 2-tuple. The first entry, the :class:`db.Source`
+    :returns: A 2-tuple. The first entry, the :class:`models.Source`
     initialized. The second, their codename string.
     """
     # Create source identity and database record
     codename = crypto_util.genrandomid()
     filesystem_id = crypto_util.hash_codename(codename)
     journalist_filename = crypto_util.display_id()
-    source = db.Source(filesystem_id, journalist_filename)
-    db.db_session.add(source)
-    db.db_session.commit()
+    source = models.Source(filesystem_id, journalist_filename)
+    models.db.session.add(source)
```

Comment:
Annoyingly, this should just be `db.session` and not `models.db.sessions`. End result of a `sed` gone bad. Technically, this isn't wrong, but it's not really worth changing unless there's something else too.